src/Controller/Admin/PageImportController.php

```
@@ -27,39 +40,80 @@ public function __construct(
-        $file = $request->files->get('file');
+        $file = $this->getUploadedFile($request);
         if (!$file instanceof UploadedFile) {
-            return new JsonResponse(['success' => false, 'message' => 'No file uploaded'], 400);
+            return $this->createJsonResponse(false, $this->messagesMap[self::ERR_NO_FILE_UPLOADED], 400);
         }
 
         $overwrite = $request->request->getBoolean('overwrite');
 
         try {
-            $page = $this->pageImporter->parseYaml($file->getContent());
+            $pages = $this->pageImporter->import($file->getContent(), $request->query->get('format', 'yaml'));
```

PHPStan (2.0.3)

Parameter #2 $format of method Neusta\Pimcore\ImportExportBundle\Documents\Import\PageImporter::import() expects string, bool|float|int|string|null given.


(argument.type)